packages/embark/src/lib/modules/codeRunner/vm.ts

```diff
@@ -33,7 +33,7 @@ class VM {
    */
   private _options: NodeVMOptions = {
     require: {
-      builtin: ["path", "rxjs", "util"],
+      builtin: ["path", "util"],
```

Comment:
Does this actually work? Builtin's are meant for node-native modules. We could add `rxjs` to the `external` property instead, which whitelists `rxjs`. Alternatively, we could add `rxjs` to the sandbox and pre-load it with an object. I believe that since `rxjs` is used in `embarkjs`, adding it to the `external` list of whitelisted modules would be the best option.

Reply:
It was in the builtin list for some reason in addition to being in the external list, I only removed it from the builtin list.